training/model_management/src/run_model_preprocess.py

```diff
@@ -12,7 +12,9 @@
 from azureml.model.mgmt.processors.pyfunc.vision.config import Tasks
 from pathlib import Path
 import shutil
+from applicationinsights import TelemetryClient
 
+tc = TelemetryClient("71b954a8-6b7d-43f5-986c-3d3a6605d803")
```

Comment:
Let us initialize only one `TelemetryClient` per package in a single place as it creates buffers for each instance.

Reply:
Done